pytools/py_codegen.py

```diff
@@ -144,10 +145,14 @@ def __init__(self, module, name):
     def _initialize(self, module, name):
         self.module = module
         self.name = name
-        self.func = module.mod_globals[name]
+        if sys.version_info < (3, 9):
+            self.func = module.mod_globals[name]
+        else:
+            self.__call__ = module.mod_globals[name]
```

Comment:
According to
https://docs.python.org/3/reference/datamodel.html#special-lookup
This shouldn't really work?